SpatialGDK/Source/SpatialGDK/Private/SpatialView/ViewDelta.cpp

```diff
@@ -247,6 +254,12 @@ ComponentChange ViewDelta::CalculateUpdate(ReceivedComponentChange* Start, Recei
 
 void ViewDelta::ProcessOp(Worker_Op& Op)
 {
+	bool bEventTracerEnabled = EventTracer != nullptr && EventTracer->IsEnabled();
+	if (bEventTracerEnabled)
+	{
+		EventTracer->DropOldUpdates();
```

Comment:
This function is called per op

Reply:
Added DropOldSpanIds into SetFromOpList for now. Good spot.